pkg/Microsoft.Private.CoreFx.UAP/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      uap-Windows_NT;
```

Comment:
Yes, the plan is to build:
 - 1 identity package
 - 4 uap runtime - specific packages (x86, x64, arm, arm64)
 - 4 uapaot runtime - specific packages (x86-aot, x64-aot, arm-aot, arm64-aot)

This PR takes care of the first two bullet points, and I will do a separate PR for the third one.